tomaty/tomaty.py

```diff
@@ -107,6 +111,16 @@ def __init__(self):
         self.notebook.append_page(
             child=self.statsPage, tab_label=Gtk.Label(label="stats"))
 
+        self.connect('delete_event', self.destory)
```

Comment:
Hey so I know that I have not been 100% consistent on this (including the original `.connect()` for the main window before your PR), but can you name your arguments passed? Given the nature of the pyGObject bindings, it's a good precaution to practice. I'm p sure [this is the documented function call](https://lazka.github.io/pgi-docs/GObject-2.0/classes/Object.html#GObject.Object.connect).

Reply:
Sorry if this is dumb, but can you clarify what you mean by this?

Reply:
rewrite `self.connect('delete_event', self.destroy)` with the formal parameter names being used from the function's signature, i.e.: 
```python
self.connect(arg_name_as_documented='delete_event', other_argument_name=self.destroy)
```

My reasoning:

`Gtk`'s bindings are filled with functions with tons of optional args which leads to various goofs, such as my goof earlier with [`timeout_add_seconds()`](https://github.com/ejmg/tomaty/commit/bed98f1f7b688d1697ad43424169e80bd19926dd#diff-a0329d9813a6b3ae7aec2f09e5fb1046R111) that led to ~4 clock speed.

Reply:
OHHHHH, okay, that makes sense. Thanks.

Reply:
actually I'm getting a `TypeError: connect() takes no keyword arguments` at the moment after fixing that lmao

Reply:
hmm, so passing them as named in the link I posted broke them?

I jumped into `gi`'s source locally and found the `.connect()` in the `Gtk.py` module. If they are not the same names you have already tried, can you try these and see if they work this time?

![screenshot from 2018-03-30 15-59-49](https://user-images.githubusercontent.com/18218174/38153698-9f0460e6-3433-11e8-947f-ae7a1b71625f.png)

Reply:
yeah- it doesn't like that.

Reply:
[_**lol**_](https://stackoverflow.com/a/24463222/5574801)

Yeah just forget about this then.

Reply:
F

Reply:
this wouldn't be as much of an issue if you had types

Reply:
Yeah, good point. That has to be a bug upstream, brb opening a PR on python core.